### PR TITLE
Remove outdated regex rules

### PR DESCRIPTION
**Pull request by orestisfl**
Context: https://github.com/i3/i3/pull/5900

### CODE REVIEW

**Comment by orestisfl** [comment]
CC @stapelberg as I can't request a review here for some reason

File: bot.go [hunk 1]
```diff
@@ -426,32 +421,20 @@ func issuesHandler(w http.ResponseWriter, r *http.Request) {
 	githubclient := github.NewClient(&http.Client{Transport: &transport})
 
 	lcBody := strings.ToLower(*payload.Issue.Body)
-	lcTitle := strings.ToLower(*payload.Issue.Title)
-
-	if *payload.Action == "opened" {
```
Comment (orestisfl) [inline bot.go:431]: Removed the `if` as that is checked here:

https://github.com/i3/i3-github-bot/blob/027111a94b3250ac01744319c69afd0c49831de5/bot.go#L412-L414

File: bot.go [hunk 2]
```diff
@@ -426,32 +421,20 @@ func issuesHandler(w http.ResponseWriter, r *http.Request) {
...
-		if bugRegexp.MatchString(lcBody) {
-			addLabel(ctx, githubclient, payload, w, "bug")
-		}
+		return
+	}
+
+	if documentationRegexp.MatchString(lcBody) {
+		// Same for documentation requests.
+		addLabel(ctx, githubclient, payload, w, "documentation")
+		return
```
Comment (orestisfl) [inline bot.go:437]: We need a new template for that in the i3 repo

**Comment by stapelberg** [comment]
Thanks! I deployed your changes just now.

**Pull request merged**